src/components/VerticalNavigation/VerticalNavigationItem.tsx

```diff
@@ -0,0 +1,113 @@
+import { Icon } from "../Icon/Icon";
+import React, { Dispatch, SetStateAction } from "react";
+import { IVerticalNavigationItem } from "./VerticalNavigation";
+
+export interface NavigationItemChild extends IVerticalNavigationItem {
+  navClick: Dispatch<SetStateAction<string[]>>;
+  selectedIds: string[];
+}
+
+/**
+ * Vertical Navigation Child Component
+ * @param {Dispatch<SetStateAction<string[]>>}     navClick       Click hanlder passed from the Veritical Navigation component that handles showing/hiding sections.
+ * @param {string[]}                               selectedIds    An array of currently selected ids that is being tracked by parent component.
+ */
+
+export const VerticalNavigationItem: React.FC<NavigationItemChild> = ({
+  navClick,
+  id,
+  selectedIds,
+  text,
+  togglable,
+  items,
+  href,
+}) => {
+  return (
+    <li className="usa-sidenav__item">
+      <a
+        href={href}
+        className={selectedIds.includes(id) ? "usa-current" : ""}
+        onClick={() =>
+          !selectedIds.includes(id)
+            ? navClick([id])
+            : togglable
+            ? navClick([])
+            : undefined
+        }
+      >
+        {text}
+        {togglable && (
+          <span className="float-right">
+            {selectedIds.includes(id) ? (
+              <Icon name="arrow_drop_up" />
+            ) : (
+              <Icon name="arrow_drop_down" />
+            )}
+          </span>
+        )}
+      </a>
```

Comment:
This block, 53-77, and 84-101 are almost identical. I feel like this is another candidate for recursion.